gio/src/tls_connection.rs

```diff
@@ -20,6 +20,11 @@ pub trait TlsConnectionExtManual {
         &self,
         type_: TlsChannelBindingType,
     ) -> Result<glib::ByteArray, glib::Error>;
+
+    #[cfg(any(feature = "v2_60", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_60")))]
+    #[doc(alias = "g_tls_connection_set_advertised_protocols")]
+    fn set_advertised_protocols(&self, protocols: IntoStrV);
```

Comment:
```suggestion
    fn set_advertised_protocols(&self, protocols: impl IntoStrV);
```

etc

Reply:
oops. I do not have 2.60 locally so I did not see those. Fixed